Description: Remove tokio-types feature & tokio dependency
 because Debian ships tokio 1.24.2
 Moreover, 1.24.2 doesn't have rt-core feature
Last-Update: 2023-02-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -42,15 +42,6 @@
 version = "1.4.2"
 optional = true
 
-[dependencies.tokio]
-version = "0.2.22"
-features = [
-    "rt-core",
-    "sync",
-]
-optional = true
-default-features = false
-
 [features]
 default = ["std"]
 detailed = [
@@ -65,4 +56,3 @@
     "std",
 ]
 std = []
-tokio-types = ["tokio"]
